src/Oauth1.php

```diff
@@ -203,7 +203,7 @@ protected function createBaseString(RequestInterface $request, array $params)
     {
         // Remove query params from URL. Ref: Spec: 9.1.2.
         $url = $request->getUri()->withQuery('');
-        $query = http_build_query($params, '', '&', PHP_QUERY_RFC3986);
+        $query = \GuzzleHttp\Psr7\build_query($params, PHP_QUERY_RFC3986);
```

Comment:
This function is now deprecated.

Reply:
@GrahamCampbell, the funtion is deprecrated in Guzzle Guzzle PSR 7.2, however this package is still supporting Guzzle 6.x. Also, the issue is not fixed by closing. In case this package would only support Guzzle PSR 7.2, I can provide a patch using the new Query::build() method.